pages/games/[id]/index.js

```diff
@@ -6,13 +6,9 @@ export default function DetailsPage() {
   const router = useRouter();
   const { id } = router.query;
 
-  const {
-    data: game,
-    isLoading,
-    error,
-  } = useSWR(`/api/games?endpoint=/thing?id=${id}`);
+  const { data: game, isLoading, error } = useSWR(`/api/games/${id}`);
```

Comment:
Since when you hard reload the detail page, the `id` will be undefined on the first render run, you can prevent an empty fetching like:

```js
useSWR(id ? `/api/games/${id}` : null);
```